README.md

```diff
@@ -83,19 +83,28 @@ and starts listening on port 53 (override with `-dns.port`), it should show the
 
 ~~~ txt
 .:53
-CoreDNS-1.6.6
+CoreDNS-1.6.9
 linux/amd64, go1.13.5, aa8c32
 ~~~
 
+To query port 53 use the following:
```

Comment:
Change to:

The following could be used to query the CoreDNS server that is running now: